features/section_explicit_waiting.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Waiting for a Section
+
+  I want to be able to explicitly wait for a section
+  In order to interact with the element once it is ready
+
+  Background:
+    When I navigate to the home page
+
+  Scenario: Wait for Section - Positive - Overridden Timeout
+    When I wait for the section that takes a while to appear
+    Then the slow section appears
+
+  Scenario: Wait for Section - Negative - Overridden Timeout
```

Comment:
The Scenario description should be more behavioral written rather than described like a unit test.

Reply:
Agreed, however all of the scenario names are now standardised, so we can tell we're covering all of the use cases